src/inference/src/threading/ie_executor_manager.cpp

```diff
@@ -7,7 +7,9 @@
 #include "ie_parallel.hpp"
 #include "threading/ie_cpu_streams_executor.hpp"
 #if IE_THREAD == IE_THREAD_TBB || IE_THREAD == IE_THREAD_TBB_AUTO
-#    include <tbb/task_scheduler_init.h>
+#    if (TBB_INTERFACE_VERSION < 12000)
```

Comment:
maybe can add some comment to illustrate this is a WA & will be fixed in the future.

Reply:
OpenVINO doesn't migrate to oneTBB so far, so we can hold about how oneTBB terminate tbb threads.
In fact, oneTBB has the different interface to terminate tbb thread, it is tbb::global_control class.